source/leuven/services/openondemand.rst

```diff
@@ -141,47 +149,68 @@ the folder that it is associated with.
-explained in the specific paragraph about the app. A general overview of the others can be found below. A more detailed guide on how to choose your resources
-is available in the next chapter.
-
-- Account: the credit account you want to deduct the credits from. The accounts associated with your VSC number will be displayed in a dropdown.
-- Partition: you can choose any of the existing partitions on both clusters. We recommend using the ``interactive`` partition for most interactive work on wICE. Be aware that this partition is not available on Genius. There it is recommended to just request the regular ``batch`` partition (see the :ref:`Choosing your resources<choosing_your_resources>` section for more detail on how to choose your partition).
+This menu provides a range of different apps that provide a GUI.
+In the background this means that you are submitting an interactive job to the cluster, in which the app will be running.
+
+To launch any of the interactive apps, you need to fill in the resources form.
+Most of the options in the resource forms are similar across all apps, but some apps require additional input from the user.
+These will be explained in the specific paragraph about the apps.
+A more detailed guide on how to choose your resources is available in the next chapter.
+Beware that by launching any app you will end up in a regular queue, so requesting a large amount of resources might result in a long queue time.
+
+- Cluster: allows choosing between one of our :ref:`Tier-2 clusters <kul_tier2>` in production, namely Genius or wICE
+- Account: the credit account you want to deduct the credits from.
+  The accounts associated with your VSC number will be displayed in a dropdown menu.
+- Partition: you can choose any of the existing partitions on both clusters.
+  The partition names depend on your choice of cluster.
+  We recommend using the ``interactive`` partition for most interactive work.
 - Numbers of hours: your walltime (min 1h).
 - Number of cores: the amount of cores per node. This defaults to 1.
 - Required memory per core in megabytes. This defaults to 3400 MB.
-- Number of GPUs. Depending on the partition you have requested, you might get a different GPU. The acquired GPU will be the same as the type specified in the partition (e.g. a NVidia V100 for ``gpu_v100``). For wICE, you can also request a GPU from the ``interactive`` partition. One GPU here is a virtual GPU slice of the available A100 GPUs. One GPU slice is the same as 1/7th of an A100 GPU. The default is 0. You can specify the type of GPU as well: [Type]:<number> (e.g. A100:2). You can also just request a number of GPUs as <number>.**The interactive partition only allows you to request max 1 GPU (slice) though.**
+- Number of GPUs. Depending on the partition you have requested, you might get a different GPU.
```

Comment:
This is different now, as each GPU type now has its own partition. So you always get the GPU type of the partition.

Reply:
Do you want me to remove the rest of the text after the first sentence?
I think the rest of the information are still valid, no?

Reply:
The part about the interactive partition is rather relevant for ondemand though, as this is the preferred partition where users submit their interactive sessions to. So at least mentioning that you can get a GPU on the interactive partition seems important. The whole part about the type of GPU, yes I would remove that.

Reply:
The updated text already reflects your idea. But, you might want to check it again.